src/systems/scene_broadcaster/SceneBroadcaster.cc

```diff
@@ -199,9 +201,14 @@ class ignition::gazebo::systems::SceneBroadcasterPrivate
   public: std::chrono::time_point<std::chrono::system_clock>
       lastStatePubTime{std::chrono::system_clock::now()};
 
-  /// \brief Period to publish state, defaults to 60 Hz.
-  public: std::chrono::duration<int64_t, std::ratio<1, 1000>>
-      statePublishPeriod{std::chrono::milliseconds(1000/60)};
+  /// \brief Period to publish state while paused and running. The key for
+  /// the map is used to store the publish period when paused and not
+  /// paused. The not-paused (a.ka. running) period has a key=false and a
+  /// default update rate of 60Hz. The paused period has a key=true and a
+  /// default update rate of 30Hz.
```

Comment:
This is needed for StateAware spin boxes. The GUI doesn't get play/pause changes from other sources unless the scene broadcaster periodically sends sate information. It was added in #1171

Reply:
> The GUI doesn't get play/pause changes from other sources unless the scene broadcaster periodically sends sate information.

It could get it from the `/world/name/stats` topic. That's what the `WorldStats` and `WorldControl` widget listen to. That would be cheaper than sending the whole state

Reply:
I took the approach of suppressing ECM serialization when paused in a819658. This allows a user of the `/state` topic to get the simulation state while paused without having to subscribe to two different topics.

Reply:
Thanks, I think this works.